appserver/ejb/ejb-container/src/main/java/com/sun/ejb/EjbInvocation.java

```diff
@@ -282,6 +264,38 @@
     // True if lock is currently held for this invocation
     private boolean holdingSFSBSerializedLock = false;
 
+    private int interceptorIndex;
+
+    public Method beanMethod;
+
+    // Only set for web service invocations.
+    private WebServiceContext webServiceContext;
+
+    // Only set for EJB JAXWS
+    // FIXME: private Message message = null;
+    private Object message;
```

Comment:
I said overlapping? Somebody tried to upgrade it, but ... ended in half of the path. Don't continue without tests, because it has processing stages.